transformations/close_homophones_swap/README.md

```diff
@@ -0,0 +1,29 @@
+# Close Homophones Swap 🦎  + ⌨️ → 🐍
+This perturbation adds perturbation to sentences and paragraphes mimicing writing behaviors where users swap words with similar homophones either intentionally or by accident.
+
+Author name: Kaizhao Liang
+Author email: kl2@illinois.edu
+Author Affiliation: University of Illinois, Urbana Champaign
+
+## What type of a transformation is this?
+This transformation acts like a perturbation to test robustness. Few words picked at random are replaced with words with similar homophones
+which sound similar or look similar. Generated transformations display high similarity to the 
+source sentences i.e. the code outputs highly precise generations. 
+
+## What tasks does it intend to benefit?
+This perturbation would benefit all tasks which have a sentence/paragraph/document as input like text classification, 
+text generation, etc. 
+
+## Previous Work
+1) There has been some recent work studying how to create human-like behavior using phonetic information:
+```bibtex
+@article{eger2019text,
+  title={Text processing like humans do: Visually attacking and shielding NLP systems},
```

Comment:
This is the right author but not the right paper.

Reply:
Hi @samsontmr, the augmentation was indeed inspired by this paper. Maybe one of the citations in this paper is more relevant, where they actually swapped out similarly sounding words to bypass censorship. Do you have another paper in mind?

Chaya Hiruncharoenvate, Zhiyuan Lin, and Eric Gilbert. 2015. Algorithmically bypassing censorship on sina weibo with nondeterministic homophone substitutions. In Proceedings of the Ninth International Conference on Web and Social Media,
ICWSM 2015, University of Oxford, Oxford, UK,
May 26-29, 2015, pages 150–158.

Reply:
I had this paper in mind actually (by the same author): https://aclanthology.org/2020.aacl-main.79/

Reply:
Thanks! this one indeed looks more relevant. Let me update the reference.

Reply:
Thanks, saw the update. Could you use the ACL bibtex instead of the arxiv? You can get it from the link I sent.

Reply:
Additionally, if you would like to implement the Chinese version (from the ICWSM paper) that would be great! You can either extend it in this PR or a new one, both ways are fine :)